docs/using.md

```diff
@@ -60,6 +60,21 @@ Another common example might be the URL of the ISO to be used. Usually you speci
 _iso-url_ of the [start](./docs/minishift_start.md) command. Applying the rules from above, you can
 also specify this URL by setting the environment variable _MINISHIFT_ISO_URL_.
 
+Minishift also maintain a config file (_~/.minishift/config/config.json_) which can be used to set some
+variable like (CPU, memory ...etc.) and can be used without using different parameter in start command.
```

Comment:
> and can be used without using different parameter in start command

". This can be used to control some of the default behavior similar to using environment variables. "

Also link to the environment variable section